llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1611,6 +1648,29 @@ void VPReductionRecipe::print(raw_ostream &O, const Twine &Indent,
     O << " (with final reduction value stored in invariant address sank "
          "outside of loop)";
 }
+
+void VPReductionEVLRecipe::print(raw_ostream &O, const Twine &Indent,
+                                 VPSlotTracker &SlotTracker) const {
+  O << Indent << "REDUCE ";
```

Comment:
Can you add a test for printing the recipe, if there isn't one yet? (Couldn't find one)

Reply:
8f48074fc7b4a8289653fcc6448420443b355a3f
Sure, add the test case for printing the recipe.